packages/code/src/Code/Code.tsx

```diff
@@ -115,12 +128,20 @@ function Code({
     }
   }, []);
 
+  useIsomorphicLayoutEffect(() => {
+    if (!expandable) return;
+
+    const codeContent = document.getElementById('code-content');
```

Comment:
Is it possible to use a `ref` here instead of an `id`?

Reply:
Yup, definitely is, and it's helpful for calculating the height now that we'll be transitioning. Great point!